src/S3/MalformedResponseParser.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace Aws\S3;
+
+use Aws\Api\Parser\AbstractParser;
+use Aws\CommandInterface;
+use Aws\Exception\AwsException;
+use Exception;
+use Psr\Http\Message\ResponseInterface;
+
+class MalformedResponseParser extends AbstractParser
```

Comment:
So is this parser for when S3 returns a ListObjects response that is invalid XML?